### ISSUE TITLE
GroupedList : GroupedList child components are not accessibile using tab/keyboard

### ISSUE THREAD

**Issue by ankurkanika04** [microsoft/fluentui#21823]
Fluent ui version - 8.49.6
React - 16.8.6

In the attached link, if i enable grouped list the child component are not accessible through keyboard navigation and flow seems to be broken.
Removing groupedlist and adding plain header by uncommenting commented code in render method seems to be working fine. So its an issue with grouped list keyboard navigation.

https://codesandbox.io/s/clever-haze-f6025l?file=/src/index.tsx

Actual Behavior - 
Grouped list child component are not properly accessible through keyboard navigation and focus is stuck and also not working properly.

Expected behavior :
Grouped list child component should be properly accessible through keyboard navigation using tabs/arrow keys

**Comment by SillyKeith** [comment]
Any update to when this will be fixed?  Our group is also hitting this issue.
